### ISSUE TITLE
libsgx-enclave-common can no longer be installed on some systems

### ISSUE THREAD

**Issue by jethrogb** [intel/linux-sgx#389]
Correct installation (version 2.4.100.48163-xenial1): https://travis-ci.com/fortanix/rust-sgx/builds/106778487 (see under “Adding APT Sources”)
Failed installation (version 2.5.100.49891-xenial1): https://travis-ci.com/fortanix/rust-sgx/builds/106859391

**Comment by jethrogb** [comment]
Easy reproduction with vagrant:
```
# Vagrantfile

# See https://github.com/hashicorp/vagrant/issues/9442
Vagrant::DEFAULT_SERVER_URL.replace('https://vagrantcloud.com')

Vagrant.configure("2") do |config|
  config.vm.box = "ubuntu/xenial64"
  config.vm.provision :shell, inline: <<~SCRIPT
    echo 'deb https://download.01.org/intel-sgx/sgx_repo/ubuntu xenial main' >> /etc/apt/sources.list
    curl https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add -
    apt-get update
    apt-get install -y --no-install-recommends build-essential
SCRIPT
end
```

```
vagrant up
vagrant snapshot save provisioned
# Fails
vagrant ssh -- sudo apt-get install -y --no-install-recommends libsgx-enclave-common
vagrant snapshot restore provisioned
# Works
vagrant ssh -- sudo apt-get install -y --no-install-recommends libsgx-enclave-common=2.4.100.48163-xenial1
```

**Comment by jethrogb** [comment]
Looks like the 2.5 Debian package wasn't built correctly. This line is missing:
```
Depends: libc6 (>= 2.15), libcurl3 (>= 7.16.2), libgcc1 (>= 1:3.0), libprotobuf9v5, libssl1.0.0 (>= 1.0.1), libstdc++6 (>= 5.2)
```

**Comment by jethrogb** [comment]
Introduced in https://github.com/intel/linux-sgx/commit/d10cabebb5512878e84f5d21cdf27c39c428ffe2#diff-32232004aedaf4b107f5e0023a267d9a

**Comment by andyzyb** [comment]
Can you try the latest release 2.5.1?

**Comment by jethrogb** [comment]
Fixed in b46e0b0003ed12e7140e0d02f9ead1fb6c7d38c0
